pynicotine/config.py

```diff
@@ -702,6 +702,26 @@ def _set_config(self):
                 # Set default value
                 self.sections[section][option] = value
 
+        # Map legacy folder/user grouping modes (3.1.0)
+        for (section, option) in [
+            ("searches", "group_searches"),
+            ("transfers", "groupdownloads"),
+            ("transfers", "groupuploads")
+        ]:
```

Comment:
```suggestion
        for section, option in (
            ("searches", "group_searches"),
            ("transfers", "groupdownloads"),
            ("transfers", "groupuploads")
        ):
```

Small nit. This is the code style we use elsewhere nowadays.